frontend/src/components/skeletons/post-card-skeleton.tsx

```diff
@@ -2,16 +2,24 @@ import { Skeleton } from '@/components/ui/skeleton';
 
 export const PostCardSkeleton = () => {
   return (
-    <div className="w-full md:w-1/2 lg:w-1/3 xl:w-1/4">
-      <div className="mr-8 mt-4 mb-4 rounded-lg bg-light shadow-md dark:bg-dark-card">
-        <Skeleton className="h-48 w-full rounded-lg bg-slate-200 dark:bg-slate-700" />
+    <div className="w-full sm:w-1/2 md:w-1/2 lg:w-1/3 xl:w-1/4">
+      <div className="mb-4 rounded-lg bg-light shadow-md dark:bg-dark-card">
+        <Skeleton className="h-48 sm:h-64 w-full rounded-lg bg-slate-200 dark:bg-slate-700" />
         <div className="p-4">
-          <Skeleton className="mb-2 h-3 w-2/3 bg-slate-200 dark:bg-slate-700" />
-          <Skeleton className="mb-2 h-6 w-4/5 bg-slate-200 dark:bg-slate-700" />
-          <Skeleton className="h-16 w-11/12 bg-slate-200 dark:bg-slate-700" />
+          <Skeleton className="mb-2 h-3 w-full sm:w-2/3 bg-slate-200 dark:bg-slate-700" />
+          <Skeleton className="mb-2 h-6 w-full sm:w-4/5 bg-slate-200 dark:bg-slate-700" />
+          <Skeleton className="h-16 w-full sm:w-11/12 bg-slate-200 dark:bg-slate-700" />
           <div className="mt-2 flex flex-wrap gap-2">
```

Comment:
These looks perfect, but just cross verify if those both are similar in sizes, for confirmation...